### ISSUE TITLE
Grouped actions should display run status before expanding them.

### ISSUE THREAD

**Issue by megamax34** [Azure/LogicAppsUX#6708]
### Describe the issue

- Open up the new logic app ui
- Create an action that groups other actions (condition, switch, scope, etc)
- Lets use the switch action for this example.
- Create a couple different switch cases that each have their own actions.
- Run the logic app so one of the cases is hit. 
- Open up the run and go to the switch statement. 
- It will have a green check 
- Expand the switch to see the cases. 
- At this point we have no idea which case was executed, skipped, or failed. 
- There needs to be an indicator like a green check bubble or red x just like there is for every other action.
- To figure out which switch case was executed you have to open them all up till you find it. 
- Yes you could click on the switch, wait for the action to load, read the value, and hope you labeled your cases in a way you can figure it out without opening them. 
- Please refer to screen shot below.

![Image](https://github.com/user-attachments/assets/3bad5d7c-e2ba-465f-9bb5-cad7d7bea9f4)

### Are you using new designer or old designer

New Designer (With action panels)

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/cbffe791-67f3-4929-95c5-187ac386a958)

### Browser

edge

**Comment by ccastrotrejo** [comment]
Hi @megamax34 thanks for raising this issue. This looks more like a feature request rather than an accessibility issue. I will take a look on how we can implement this. In the meantime, as you said, please keep expanding the actions as you said. Also, we introduced a new gesture in which you can quickly uncollapse everything under the switch action with a right click: https://github.com/Azure/LogicAppsUX/pull/6494.

**Comment by megamax34** [comment]
Thank you Carlos for getting that into the right state! Also this new
update you linked is awesome, please pass my appreciation to the team!
Thanks again!!
